src/python.cpp

```diff
@@ -1980,8 +1980,23 @@ PyObjectRef py_dict_impl(const List& keys, const List& items, bool convert) {
 
 // [[Rcpp::export]]
 SEXP py_dict_get_item(PyObjectRef dict, RObject key) {
-  PyObjectPtr pyKey(r_to_py(key, dict.convert()));
+  
+  PyObject* pyKey(r_to_py(key, dict.convert()));
```

Comment:
Doesn't changing this to `PyObject*` mean we risk leaking a reference to the Python object here?

Reply:
no comment ;-) [--> goes hang herself]]